gnocchi/storage/redis.py

```diff
@@ -94,7 +94,7 @@ def _delete_metric(self, metric):
 
     def _get_measures(self, metric, keys, aggregation, version=3):
         if not keys:
-            return []
+            return {}
```

Comment:
is there a reason you change this to return empty dict? this is just serialized data so i don't think it's an dict... or a list either

Reply:
It's actually a list that is returned in the canonical implementation in `gnocchi.storage`. I'll fix it.